documentation/developers.md

```diff
@@ -0,0 +1,39 @@
+## TL;DR Quick Start
+
+1. Add `puppetlabs/trapperkeeper-status` to your lein deps.
+2. Add `puppetlabs.trapperkeeper.services.status.status-service/status-service`
+   to your `bootstrap.cfg`.
+3. Add `[:StatusService register-status]` to your TK service's deps.
+4. Call `status-core/get-artifact-version` to dynamically get the
+   version info for your service.
+5. Call `register-status` to register a callback function that returns
+   status info for your service.
+6. Define your status callback function.
+
+Code sample:
+
+    (ns foo
```

Comment:
Tiny thing: I think we can use \`\`\`clojure here to get clojure syntax highlighting, although I think that only works on github.